rosidl_typesupport_c/src/action_type_support_dispatch.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 Open Source Robotics Foundation, Inc.
```

Comment:
Same as above comments, I don't think this file is needed.

Reply:
True.